src/calendar/stories/Calendar.stories.tsx

```diff
@@ -3,22 +3,16 @@ import { Meta } from "@storybook/react";
 import { addDays, addWeeks, subWeeks } from "date-fns";
 
 import "./index.css";
+import { DateValue } from "../index";
```

Comment:
import from "index.d" just to be specific that we are importing from the d.ts file